src/scheduler/queue_info.c

```diff
@@ -173,9 +175,11 @@ query_queues(status *policy, int pbs_sd, server_info *sinfo)
 	}
 	qinfo_arr[0] = NULL;
 
+	my_partitions = break_comma_list(partitions);
```

Comment:
Like above, you probably want to check for NULL

Reply:
Done